exonum-java-binding/core/rust/Cargo.toml

```diff
@@ -13,6 +13,7 @@ crate-type = ["dylib"]
 # Enables native handles checking. Useful for debugging.
 resource-manager = []
 invocation = ["jni/invocation"]
+default = ["resource-manager", "invocation"]
```

Comment:
It apparently breaks unit tests (they are all with rm now):

```
[INFO] --- exec-maven-plugin:1.6.0:exec (test-rust-lib) @ exonum-java-binding-core ---
   Compiling exonum-java v0.7.0-SNAPSHOT (/home/travis/build/exonum/exonum-java-binding/exonum-java-binding/core/rust/exonum-java)
    Finished dev [unoptimized + debuginfo] target(s) in 1m 14s
     Running target/debug/deps/exonum_java-9ab214bf4be122ec

test node_builder::tests::prepare_service_factories_missing_user_services ... ok
test result: ok. 7 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
     Running target/debug/deps/java_bindings-d87b051662c9dc28
running 36 tests
test handle::resource_manager::imp::tests::add_zero_handle ... ok
test handle::resource_manager::imp::tests::check_invalid_handle ... ok
test handle::resource_manager::imp::tests::add_duplicated_handle ... ok
test handle::resource_manager::imp::tests::check_wrong_type_handle ... ok
test handle::resource_manager::imp::tests::check_zero_handle ... ok
test handle::resource_manager::imp::tests::manage_handles ... ok
test handle::resource_manager::imp::tests::remove_invalid_handle ... ok
test handle::resource_manager::imp::tests::remove_zero_handle ... ok
test handle::tests::cast_zero_object ... ok
test handle::resource_manager::imp::tests::check_wrong_ownrship_handle ... ok

test result: ok. 36 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
[INFO] 
[INFO] --- exec-maven-plugin:1.6.0:exec (test-rust-lib-no-resource-manager) @ exonum-java-binding-core ---
    Finished dev [unoptimized + debuginfo] target(s) in 0.54s
     Running target/debug/deps/java_bindings-d87b051662c9dc28
running 36 tests
test handle::resource_manager::imp::tests::add_duplicated_handle ... ok
test handle::resource_manager::imp::tests::check_invalid_handle ... ok
test handle::resource_manager::imp::tests::add_zero_handle ... ok
test handle::resource_manager::imp::tests::check_wrong_ownrship_handle ... ok
test handle::resource_manager::imp::tests::check_wrong_type_handle ... ok
test handle::resource_manager::imp::tests::check_zero_handle ... ok
test handle::resource_manager::imp::tests::manage_handles ... ok
test handle::resource_manager::imp::tests::remove_invalid_handle ... ok
test handle::resource_manager::imp::tests::remove_zero_handle ... ok
```

It is either cargo test bug, or this change requires additional flags to pass.

Reply:
Removed default features from `java_binding` Cargo.toml because one might need to build lib with no feature[s] and perhaps this looks more natural then specifying `--no-default-features` for such a build.
Also, I added features `resource-manager invocation` (instead of just `resource-manager`) for building a library at `build-rust-lib-for-tests` and `test-rust-lib`.
This gives us such abilities:
1) The `java_bindings` library gets built with features on `build-rust-lib-for-tests`.
2) Unit tests for `test-rust-lib-no-resource-manager` are built without features and don't trigger library rebuild.
3) Unit tests for `test-rust-lib` are built with `resource-manager invocation` features and also don't trigger library rebuild. There is the unit tests binary for exonum-java which formally depens on `java_binding` shared library (because `exonum-java` crate depends on it) but it is also built with these features -> no library rebuild.
4) Integration tests use `java_binding` library from step (1) avoiding rebuild.